### ISSUE TITLE
Property 'promisifyAll' does not exist on type 'typeof Bluebird'

### ISSUE THREAD

**Issue by shekhardokania** [havsar/node-ts-cache#14]
node_modules/node-ts-cache/src/storage/fs.json.storage.ts:4:21 - error TS2339: Property 'promisifyAll' does not exist on type 'typeof Bluebird'.
```javascript
4 const Fs = Bluebird.promisifyAll(require('fs'))
                      ~~~~~~~~~~~~

node_modules/node-ts-cache/src/storage/redis.storage.ts:7:10 - error TS2339: Property 'promisifyAll' does not exist on type 'typeof Bluebird'.

7 Bluebird.promisifyAll(Redis.RedisClient.prototype)
           ~~~~~~~~~~~~

node_modules/node-ts-cache/src/storage/redis.storage.ts:8:10 - error TS2339: Property 'promisifyAll' does not exist on type 'typeof Bluebird'.

8 Bluebird.promisifyAll(Redis.Multi.prototype)
```

NodeJS version: 10.16.3
typescript version: 3.4.4

The error is happening when I am trying to compile my typescript code.

**Comment by havsar** [comment]
Fix will be included in the next version (4.0.0.).

**Comment by lxandrexl** [comment]
this is solved ? cuz i have error node v14.17.0 / npm v6.14.13
please help!
